### PR TITLE
[ros] camera_info header stamp modified

### LINKED ISSUE #2980: Problem using depth images to create point cloud
Hi all, 

As mentioned in the document of AirSim [Image APIs - AirSim](https://microsoft.github.io/AirSim/image_apis/#available_cameras): 

`For example, you may be able to feed perspective depth to ROS package such as depth_image_proc to generate a point cloud`

we can use depth images from AirSim to generate XYZ point cloud, or even XYZRGB point cloud. 

However, when I fed depth image & related camera info rostopics to the depth_image_proc/point_cloud_xyz node, I got the following WARNs, and no point cloud came out:

```
[ WARN] [1598496524.956492599]: [image_transport] Topics '/depth_image' and '/depth_camera_info' do not appear to be synchronized. In the last 10s:
	Image messages received:      25
	CameraInfo messages received: 0
	Synchronized pairs:           0
```

* OS: ubuntu 16.04
* unreal 4.24.3
* AirSim: master at [bd8ee6b](https://github.com/microsoft/AirSim/tree/bd8ee6be4183c28d402e4c34cce114312da46358)
* ros: kinetic

### PR DESCRIPTION
**Pull request by GimpelZhang**
Fixes: #2980

The header stamps of camera_info rostopics are modified. So timestamps of image topics and related camera_info topics are the same. 

Issue described here: #2980

### CODE REVIEW

**Comment by madratman** [comment]
Hmm, I agree. Per logic, I think camera tf's timestamps should also be changed?

File: ros/src/airsim_ros_pkgs/include/airsim_ros_wrapper.h [hunk 1]
```diff
@@ -252,15 +252,15 @@ class AirsimROSWrapper
...
     void publish_odom_tf(const nav_msgs::Odometry& odom_msg);
 
     /// camera helper methods
     sensor_msgs::CameraInfo generate_cam_info(const std::string& camera_name, const CameraSetting& camera_setting, const CaptureSetting& capture_setting) const;
     cv::Mat manual_decode_depth(const ImageResponse& img_response) const;
 
-    sensor_msgs::ImagePtr get_img_msg_from_response(const ImageResponse& img_response, const ros::Time curr_ros_time, const std::string frame_id);
-    sensor_msgs::ImagePtr get_depth_img_msg_from_response(const ImageResponse& img_response, const ros::Time curr_ros_time, const std::string frame_id);
+    sensor_msgs::ImagePtr get_img_msg_from_response(const ImageResponse& img_response, const std::string frame_id);
+    sensor_msgs::ImagePtr get_depth_img_msg_from_response(const ImageResponse& img_response, const std::string frame_id);
```
Comment (rajat2004) [inline ros/src/airsim_ros_pkgs/include/airsim_ros_wrapper.h:263]: The arg can be made as reference - `const std::string& frame_id`

Comment (GimpelZhang) [reply ros/src/airsim_ros_pkgs/include/airsim_ros_wrapper.h]: I agree. References are better.

**Comment by zimmy87** [comment]
It seems that you have a lot of conflicts. This is most likely due to conflicts with autoformatted code in master. Please follow these steps to resolve this:

1. check out your pull request
2. run `git checkout master .clang-format` in your repo's root folder
3. if you don't have npm installed, download it from https://www.npmjs.com/get-npm
4. run `npm install -g clang-format`
5. run `clang-format -i path/to/file` for each conflicting file
6. run `git add -u` to track all style changes
7. run `git commit -am "apply style from clang-format"`
8. run `git merge master`
9. resolve other non-style conflicts if you have any
10. push changes

Thanks for helping us with these style changes!

**Comment by jonyMarino** [comment]
Thanks @GimpelZhang for your contribution! keep it up!

**Pull request merged**